server/src/lint.ts

```diff
@@ -16,9 +16,18 @@ export default async function lint(
     return JSON.parse(result)
   } catch (errors) {
     const errorMessage = "prisma-fmt error'd during linting.\n"
+
     if (onError) {
-      onError(errorMessage + errors)
+      if ((errors as string) && errors.includes('ParsingError')) {
+        // recognized Prisma 1 schema
+        onError(
+          "This file is a GraphQL file and is not supported by the Prisma extension. Please install the GraphQL extension ('https://marketplace.visualstudio.com/items?itemName=Prisma.vscode-graphql') from the marketplace and change the file extension to '.graphql'.",
+        )
+      } else {
```

Comment:
An invalid Prisma 2 schema would also cause a `ParsingError` right? How do we know for sure that it is indeed a Prisma 1 schema?

Reply:
We should probably be a bit more precise here and make sure it _actually_ is a GraphQL/Prisma1 schema.

Reply:
No, an invalid Prisma 2 schema would not cause a `ParsingError`, it would cause a LinterError instead.

Reply:
I'm not aware of any other situation, where a `ParsingError` would be thrown. Also I see no other way to make sure it acutally is a GraphQl/Prisma1 schema.

Reply:
Checking if `type` exists for example would increase our confidence level that this is indeed P1/GraphQL.

> No, an invalid Prisma 2 schema would not cause a ParsingError, it would cause a LinterError instead.

This confuses me. Why can only P1/GraphQL files cause a `ParsingError`?

Reply:
I'm not sure if only P1/GraphQL files cause a `Parsing Error`, but I also don't know of any other possible input causing a `ParsingError` instead of a `LinterError`. 

Sure, I can check if  `type` exists.